scripts/release/const.ts

```diff
@@ -0,0 +1,4 @@
+export const base = '../../packages/';
```

Comment:
path.resolve(__dirname, './../packages')

Reply:
and uppercase if they are  constant

Reply:
also, you have `types.ts` file that contains similar content actually. maybe mixt these two files together and call them `constants.ts`?

Reply:
Sure